content/en/getting_started/synthetics/browser_test.md

```diff
@@ -11,84 +11,77 @@ further_reading:
 
 The example below demonstrates the creation of a browser test that maps a user's journey from adding an item to a cart to successfully checking out. 
 
 {{< img src="getting_started/synthetics/browser-test.png" alt="Browser test"  style="width:90%;" >}}
 
-Datadog records every test execution as a **Test Result**.
-
 ### Configure your test details
 
-1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **Synthetic Test**.
+1. In the Datadog application, hover over **[UX Monitoring][2]** in the left hand menu and select **[Synthetic Tests][2]**.
 2. In the top right corner, click the **New Test** button.
 3. Select **Browser Test**.
 4. Define your browser test:
 
     - Add the URL of the website you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io`, a test web application.
-    - Select **Advanced Options** to use custom request headers, authentication credentials, proxy URLs, or cookies. 
-    - Set environment tags, for example `env:prod` and `app:shopist`, and additional tags for your test. Tags allow you to stay organized and quickly find tests you're interested in on the homepage.
+    - Select **Advanced Options** to use custom request headers, authentication credentials, or cookies. 
+    - Name your test and set tags to it such as `env:prod` and `app:shopist`. Tags allow you to keep your test suite organized and quickly find tests you're interested in on the homepage.
     - Choose the browsers and devices you want to test with. 
 
 #### Select locations
 
 Select one or more **Managed Locations** or **Private Locations** to run your test from.
 
-Your managed locations often include public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select **Private Locations** instead.
+Managed locations allow you to test public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select one of your **Private Locations** instead.
 
-For more information, see [Getting Started with Private Locations][10].
+For more information on how to set up private locations, see [Getting Started with Private Locations][3].
 
 #### Specify test frequency
 
 Select the frequency that your test runs.
 
 #### Define alert conditions
 
-Select the circumstances in which you want a test to send notification alerts.
+You can define alert conditions to ensure your test does not trigger in case of sporadic network blip and that you only get alerted in case of real issues with your application.
```

Comment:
```suggestion
You can define alert conditions to ensure your test does not trigger for things like a sporadic network blip, so that you only get alerted in case of real issues with your endpoint.
```